src/AspNetCore/Src/forwardinghandler.cxx

```diff
@@ -1397,9 +1398,11 @@ FORWARDING_HANDLER::OnExecuteRequestHandler(
             &cchANCMHeader))) // first time failure
         {
             if (SUCCEEDED(hr = m_pW3Context->CloneContext(
-                CLONE_FLAG_BASICS | CLONE_FLAG_HEADERS | CLONE_FLAG_ENTITY,
-                &m_pChildRequestContext
-            )) &&
+                    CLONE_FLAG_BASICS  |
+                    CLONE_FLAG_HEADERS |
+                    CLONE_FLAG_ENTITY  |
+                    CLONE_FLAG_SERVER_VARIABLE,
```

Comment:
Why do we need to add clone server variables?

Reply:
we should. It was missing because we targeted on Vista in old depot which does not have this option.